oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHWDiskStore.java

```diff
@@ -270,6 +300,27 @@ private static void computeDiskStats(LinuxHWDiskStore store, String devstat) {
         store.transferTime = devstatArray[UdevStat.ACTIVE_MS.ordinal()];
     }
 
+    private static String getPartitionNameForDmDevice(String vgName, String lvName) {
+        return new StringBuilder().append(DEV_LOCATION).append("/").append(vgName).append("/").append(lvName).toString();
+    }
+
+    private static String getMountPointOfDmDevice(String vgName, String lvName) {
+        return new StringBuilder().append(DEV_MAPPER).append("/").append(vgName).append("-").append(lvName).toString();
+    }
+
+    private static String getDependentNamesFromHoldersDirectory(String sysPath) {
+        File holdersDir = new File(sysPath + "/holders");
+        File[] holders = holdersDir.listFiles();
+        StringBuilder dependentNames = new StringBuilder();
```

Comment:
Consider a `StringJoiner` here to avoid having to deal with that extra delimiter.   `new StringJoiner(" ");` and then `.add()` the `f.getNames()` to it.  Or, alternately, `return Arrays.stream(holders).map(f -> f.getName()).collect(Collectors.joining(" "));`

Reply:
your way is much better than mine :)

Reply:
Actually it's even better with a lambda:
```
Arrays.stream(holders).map(File::getName).collect(Collectors.joining(" "));
```